tests/unit/rules/javascript/test.no_implied_eval.js

```diff
@@ -80,8 +82,8 @@ describe('no_implied_eval', () => {
       description: [NO_IMPLIED_EVAL.description],
     },
     {
-      code: 'execScript("x = 1;");',
-      message: [expectedErrorMessage],
+      code: 'window.execScript("x = 1;");',
```

Comment:
I had to change this because of https://github.com/eslint/eslint/issues/19923, but I am not entirely sure that's what we want.